### PR TITLE
Update the version of Perfetto

### PR DESCRIPTION
**Pull request by AWoloszyn**

### CODE REVIEW

File: core/cc/android/process_name.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ */
+
+#include <fstream>
+#include <streambuf>
+#include <string>
+
+namespace core {
+static const size_t MAX_PATH = 4096;
+
+std::string get_process_name() {
```
Comment (hevrard) [inline core/cc/android/process_name.cpp]: This looks unrelated to perfetto?

File: core/cc/osx/process_name.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+#include <libgen.h>
+#include <unistd.h>
+#include <cstring>
+#include <string>
+
+namespace core {
+static const size_t MAX_PATH = 4096;
+
+std::string get_process_name() {
```
Comment (hevrard) [inline core/cc/osx/process_name.cpp]: unrelated?

File: test/integration/gles/replay/BUILD.bazel [hunk 3]
```diff
@@ -62,8 +62,9 @@ go_test(
     ],
     data = [
         "//cmd/gapir/cc:gapir",
+        "//core/vulkan/vk_api_timing_layer/cc:json",
+        "//core/vulkan/vk_api_timing_layer/cc:libVkLayer_APITiming",
```
Comment (hevrard) [inline test/integration/gles/replay/BUILD.bazel]: unrelated?

File: test/integration/replay/BUILD.bazel [hunk 4]
```diff
@@ -26,8 +26,8 @@ go_test(
     srcs = ["postback_test.go"],
     data = [
         "//cmd/gapir/cc:gapir",
+        "//core/vulkan/vk_api_timing_layer/cc:libVkLayer_APITiming",
```
Comment (hevrard) [inline test/integration/replay/BUILD.bazel]: unrelated?

File: test/integration/gles/replay/BUILD.bazel (hunk 3 above)
Comment (AWoloszyn) [reply test/integration/gles/replay/BUILD.bazel]: I broke the build with my last CL so this fixes it as well. (build == tests)
